tests/modules/blockonkeys.c

```diff
@@ -546,6 +596,9 @@ int RedisModule_OnLoad(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
     if (RedisModule_CreateCommand(ctx,"fsl.push",fsl_push,"write",1,1,1) == REDISMODULE_ERR)
         return REDISMODULE_ERR;
 
+    if (RedisModule_CreateCommand(ctx,"fsl.pushtimer9000",fsl_pushtimer9000,"write",1,1,1) == REDISMODULE_ERR)
+        return REDISMODULE_ERR;
```

Comment:
why encode the value in the command name? why not `fsl.pushtimer` that takes an argument?
don't have a problem with it, just that it's ugly.